perfdash/config.go

```diff
@@ -340,6 +340,136 @@ var (
 					Parser:           parsePerfData,
 				}},
 		},
+		"Network_Baseline": {
+			"TCP_P2P": []TestDescription{{
+				Name:             "standalone",
+				OutputFilePrefix: "NetworkPerformanceMetrics1:1TCPP2P",
```

Comment:
Can we have better names (it's hard to parse it), e.g.:

NetworkPerformanceMetrics_1:1_TCP_P2P

[Yes - it requires small changes to the code generating them]

Reply:
Also - converge on the order (the the name should be the ame as in the file prefix)

Reply:
Modified